### ISSUE TITLE
Color Contrast Not Calculating Correctly

### ISSUE THREAD

**Issue by adrianavecchiodhs** [uswds/uswds#4146]
### Describe the bug
Using the color "dhs-green", #5e9732, color contrast requires the text be black, but USWDS is displaying white text for green buttons

### Steps to reproduce the bug
1. Go to https://www.dhs.gov/xlibrary/dhsweb/_site/components.html
2. Scroll down to Buttons
3. Under "Secondary" see green buttons. Default button displays as "dhs-green" which should have black text, but shows white text

### Expected behavior
When using "dhs-green" as a background, USWDS should show black text

### Screenshots
Add screenshots to help provide context, if applicable.

### System setup
- **USWDS version:**
- **Device:**
- **Operating system:**
- **Browser and version:**

### Additional context
Add any other context about the problem.

**Comment by mejiaj** [comment]
Confirmed via browser devtools and AXE. `#ffffff` on `#5e9732` fails color contrast.
![image](https://user-images.githubusercontent.com/3385219/121954629-99560400-cd24-11eb-886f-ec1fc0996749.png)
